apertium-eng.eng.rlx

```diff
@@ -212,6 +217,54 @@ SELECT N IF (0 N) (0 Adj) (1 V) ;
 
 SELECT Adj IF (0 N) (0 Adj) (1 N) ;
 
+# Different cases for as ("as __ as")
+
+SELECT Pr IF (0 As) (-2 Preadv) (-1 Adv) (-3 Vbser) ;
+#Example (as):
+#You are as quick as I am.
+
+SELECT Pr IF (0 As) (-2 Preadv) (-1 Adv) (-4 Vbser) ;
+#Example (as):
+#The Earth was formed as early as 4.54 billion years ago.
+
+SELECT Pr IF (0 As) (-2 Preadv) (-1 Adj) (-3 Vbser) ;
+#Example (as):
+#You are as nice as I am.
+
+SELECT Pr IF (0 As) (-2 Preadv) (-1 Adj) (-4 Vbser) ;
+#Example (as):
+#You are not as evil as I am.
+
+# Has (Cnjsub) and has (Det)
+
+SELECT Cnjsub IF (0 Det) (-1 Adj) (-2 Vbser) (1 Nsg) ;
+#Example (that):
+#It is true that candy is evil.
+
+SELECT Cnjsub IF (0 Det) (-1 Vbser) (-2 N) (1 Nsg) ;
+#Example (that):
+#The problem is that candy is evil.
+
+SELECT Det IF (0 Cnjsub) (1 Nsg) (2 Vbser) ;
+#Example (that):
+#That cat is evil.
+
+SELECT Det IF (0 Cnjsub) (-1 Vbser) (1 Nsg) ;
```

Comment:
This appears to conflict with the rule above:
`SELECT Cnjsub IF (0 Det) (-1 Vbser) (-2 N) (1 Nsg) ;`

I'm not sure what will happen if an example matching both of them is encountered (@TinoDidriksen?), but you shouldn't be writing conflicting rules like this.

Reply:
Sure, I can edit/remove that rule.  As for two conflicting rules, I think rules that are higher up take precedence.